traversal/planner/GraphPlanner.java

```diff
@@ -179,6 +182,13 @@ private void createOptimiserConstraints() {
                 conOneVertexAtOrderI.setCoefficient(vertex.varOrderAssignment[i], 1);
             }
         }
+        for (int i = 0; i < modifiers.sorting().variables().size(); i++) {
+            Identifier.Variable.Retrievable sortVariable = modifiers.sorting().variables().get(i);
+            PlannerVertex<?> vertex = vertices.get(sortVariable);
+            assert vertex != null;
+            OptimiserConstraint conSetVertexAtOrderI = optimiser.constraint(1, 1, conPrefix + "set_vertex_at_order_" + i);
+            conSetVertexAtOrderI.setCoefficient(vertex.varOrderAssignment[i], 1);
+        }
```

Comment:
the ordering variables are pinned to the beginning of the plan. This guarantees that we generate answers in an ordered fashion